lib/views/pay/pay_view.dart

```diff
@@ -1,114 +1,255 @@
+                        margin: const EdgeInsets.fromLTRB(20, 10, 20, 10),
+                        shape: const RoundedRectangleBorder(
+                            borderRadius:
+                                BorderRadius.all(Radius.circular(20.0))),
+                        child: Center(
+                          child: Padding(
+                            padding: const EdgeInsets.all(20),
+                            child: Column(
+                              children: [
+                                Text(
+                                  invoice.invoice.pubKey,
+                                  style: TextStyle(
+                                    color: Theme.of(context).disabledColor,
+                                    fontSize: 18,
+                                  ),
+                                ),
+                                Expanded(
+                                  flex: size.width > 400 ? 1 : 2,
+                                  child: Card(
+                                    shape: RoundedRectangleBorder(
+                                      borderRadius: BorderRadius.circular(20),
+                                      side: const BorderSide(
+                                        color: Colors.white,
+                                        width: 2.0,
+                                      ),
+                                    ),
+                                    margin: const EdgeInsets.only(top: 15),
+                                    child: Padding(
+                                      padding: const EdgeInsets.all(15),
+                                      child: Center(
+                                        child: Text(
+                                          invoice.invoice.description,
+                                          style: const TextStyle(fontSize: 18),
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                ),
+                                const SizedBox(
+                                  height: 10,
+                                ),
+                                Expanded(
+                                  flex: 1,
+                                  child: Row(
+                                    mainAxisAlignment:
+                                        MainAxisAlignment.spaceBetween,
+                                    children: [
+                                      Text(
+                                        'Created time \n${invoice.invoice.createdTime}',
+                                        style: const TextStyle(fontSize: 18),
+                                      ),
+                                      Text(
+                                        'Expiration time \n${invoice.invoice.expirationTime}',
+                                        style: const TextStyle(fontSize: 18),
```

Comment:
Nobody is able to understand a UNIX timestamp :) Are you?

So why do we not put the date print as a date? 

Some docs https://stackoverflow.com/a/50632411/10854225

Reply:
Oh nice, updating right away